hw2/03-count.js

```diff
@@ -1,3 +1,35 @@
 // Add your code here
+const input = document.querySelector("input");
+let searchArray = [];
+//const mainTextArray = cleanText();
+let paragraph = document.querySelector("#paragraph");
 
-input.addEventListener('keydown', handleKeyDown);
+let text = document.getElementById("paragraph").innerHTML.toLowerCase();
+input.addEventListener("keydown", handleKeyDown);
+
+function handleKeyDown(e) {
+  if (e.key === "Backspace") {
+    e.target.value = "";
+    searchArray = [];
+  } else if (e.key === "Shift") {
```

Comment:
I'm not entirely sure why the program needs to check for shift key.  Could we just remove the if condition and just get user input + the e.key and use the `findMatch()` function to highlight the matched words?
I still don't quite understand everything in the quote so I'm not 100% sure as well.